### PR TITLE
MCS-620 New config file property to open/close container

### PR DESCRIPTION
**Pull request by kdrumm**
Updates to test open and close from config file by adding isOpen and openable to metadata
Added integration test to test open/close
Added new scene to show open and close

Associated with: https://github.com/NextCenturyCorporation/ai2thor/pull/131

### CODE REVIEW

File: machine_common_sense/object_metadata.py [hunk 1]
```diff
@@ -52,6 +52,10 @@ class ObjectMetadata(object):
         This object's colors, derived from its textures, in plain English.
     visible : boolean
         Whether you can see this object in your camera viewport.
+    isOpen : boolean
```
Comment (ThomasSchellenbergNextCentury) [inline machine_common_sense/object_metadata.py:55]: In Python, we try to keep all of the variable names in snake_case, since that's the commonly-used Python naming convention. (I know this is a little confusing since we use camelCase in Unity, since that's its naming convention for local variables, and thus we have to convert the names in scenarios like this.)

**Review by ThomasSchellenbergNextCentury — CHANGES_REQUESTED** [review]
I really appreciate the integration test and example scene file. However, I'm getting some failures/errors in the unit tests.

**Comment by kdrumm** [comment]
Updated.  Reran unit tests and integration tests with no failures on my side.

**Review by ThomasSchellenbergNextCentury — APPROVED** [review]
